cmd/genutils/genutils.go

```diff
@@ -39,3 +41,65 @@ func OutDir(path string) (string, error) {
 	outDir = outDir + "/"
 	return outDir, nil
 }
+
+func getOutDir(path string, args []string) string {
+	if len(args) == 1 {
+		path = args[0]
+	} else if len(args) > 1 {
+		fmt.Fprintf(os.Stderr, "usage: COMMAND [output directory]\n")
+		os.Exit(1)
+	}
+
+	outDir, err := OutDir(path)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to get output directory: %v\n", err)
+		os.Exit(1)
+	}
+	return outDir
+}
+
+func AddGeneratedDocsCommands(cmd *cobra.Command) {
```

Comment:
correct, added todo to convert to https://github.com/spf13/cobra/pull/143 after it lands.